packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/src/testRunOperations/testRunOperationsClient.ts

```diff
@@ -150,32 +150,48 @@ export class TestRunOperationsClient {
     testRunId: string,
     fileName: string,
     options: GetTestRunFileOptionalParams = { requestOptions: {} },
-  ): Promise<FileInfo> {
+  ): Promise<{
+    url?: string;
+    fileType?: FileType;
+    expireDateTime?: Date;
+    validationStatus?: FileStatus;
+    validationFailureDetails?: string;
+  }> {
```

Comment:
Is this expected?

Reply:
i don't think so, let me investigate there.

Reply:
should be fixed here: https://github.com/Azure/azure-rest-api-specs/pull/29521